src/routes/monitor/+page.svelte

```diff
@@ -3,16 +3,18 @@
     import MonitorChart from '$lib/components/monitorChart.svelte';
 </script>
 
-<div class="flex justify-center">
-    <h1 class="text-4xl font-bold">Terralux Kontrollstation</h1>
-</div>
+<section class="mx-auto max-w-5xl">
+    <div class="flex justify-center">
+        <h1 class="text-6xl font-bold">Terralux Kontrollstation</h1>
+    </div>
```

Comment:
Dann auch die Zentrierung einheitlich machen.
```suggestion
    <h1 class="text-center text-6xl font-bold">Terralux Kontrollstation</h1>
```